src/rpcrawtransaction.cpp

```diff
@@ -69,6 +69,7 @@ std::vector<std::pair<std::string, std::string>> GetTxStakeBoincHashInfo(const C
         res.push_back(std::make_pair(_("Magnitude"), RoundToString(claim.m_magnitude, 8)));
     }
 
+    res.push_back(std::make_pair(_("Fees Collected"), RoundToString(GetFeesCollected(block) / (double)COIN, 8)));
```

Comment:
For the GUI, we can use `FormatMoney()` here instead of `RoundToString()`. We really should use `BitcoinUnits::format()` for any money values displayed in the GUI so that the user's money unit preference is applied from `QSettings`, but `FormatMoney()` should be good enough until we have time to refactor `GetTxNormalBoincHashInfo()` and pull the logic out of the RPC area.

Reply:
done. note that FormatMoney() will show like 0.00001 with no trailing 0's fyi

Reply:
I think that should be fine for now.